src/backend/partaj/core/email.py

```diff
@@ -148,13 +148,16 @@ def send_new_message_for_requesters(cls, referral, message):
             cls.send(data)
 
     @classmethod
-    def send_referral_answered(cls, referral, answer):
+    def send_referral_answered(cls, referral, answer, targets):
```

Comment:
I think it's not ideal to have a `targets` parameter here and actually send the email to other people beyond the targets.

IMO, the best thing to do here would be to have two methods: `send_referral_answered_to_users` and `send_referral_answered_to_unit_owners`. Each method can then be in charge of finding its own contacts and then sending the emails.